src/api/udf/In.h

```diff
@@ -67,7 +66,6 @@ class In : public nebula::surface::eval::UDF<nebula::type::Kind::BOOLEAN, IK> {
         if (N_UNLIKELY(source == std::nullopt)) {
           return std::nullopt;
         }
-
```

Comment:
as mentioned above, can you put back all the empty lines, please? in all the files, just quickly go through them.